src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -331,7 +328,7 @@ class ReviewCollapsibleChapter extends React.Component {
             push
```

Comment:
Curious what the "t" is for

Reply:
Not sure, but didn't want touch it. :)

Reply:
I think it's a typo LOL... I think "push" is a typo as well.